src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1010,6 +1012,348 @@ static bool processAreaScope(Scope* scoP, BSONObj &areaQuery)
   return true;
 }
 
+/* ****************************************************************************
+*
+* qStringFilters -
+*
+* FIXME P9: this function currently abuses of the std::string() wrapping for char* in order to
+* make comparisons work. An smarter solution could be developed.
+*
+*/
+static void qStringFilters(std::string& in, std::vector<BSONObj> &filters)
+{
+  char* str         = strdup(in.c_str());
+  char* toFree      = str;
+#if 0
+  int   statementNo = 0;
+#endif
+  char* s;
+
+  while ((s = strtok(str, ";")) != NULL)
+  {
+    char*               left;
+    char*               op;
+    char*               right;
+    char*               rangeFrom = (char*) "";
+    char*               rangeTo   = (char*) "";
+    std::vector<char*>  valVector;
+
+    s = wsStrip(s);
+
+#if 0
+    printf("statement %d: %s\n", statementNo, s);
+#endif
```

Comment:
This is a way of preserving your original implementation. The idea is to keep them until feature/967 gets finished, then remove them as las task before merging feature/967 into develop.